pybikes/data/callabike.json

```diff
@@ -3,10 +3,10 @@
         {
             "tag": "callabike-aschaffenburg",
             "meta": {
-                "latitude": 49.436467,
```

Comment:
Ok at "request that is made". I ask because then it is quite possible that not all bikes are returned because of the radius, right? Also limited to 100 definitely means not all bikes are returned, at least in big cities.